initrd/bin/x230-flash.init

```diff
@@ -12,7 +12,7 @@ insmod /lib/modules/xhci-pci.ko
 insmod /lib/modules/e1000e.ko
 insmod /lib/modules/usb-storage.ko
 
-tpm extend -ix 4 -ic recovery
+tpmr extend -ix 4 -ic recovery
```

Comment:
From my understanding recovery whould need to move to a seperate file to be measured?

Reply:
This doesn't measure the recovery function, it measures the literal string `recovery` (that's what -ic does, measures the argument, -if measures a file).

Reply:
So some as for tpm1.2 got it. Was confused with other tpm2 PR attempt comments